config/locales/en.yml

```diff
@@ -40,3 +40,16 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    delete_work_notification:
+      subject: "[%{app_name}] Your work has been deleted"
+      hello: "Dear %{user},"
+      request_of: "the request of %{pseud}."
```

Comment:
The order of the sentence parts as divided here likely won't work in languages other than English. Could we have complete strings for each case?